doc/script_commands.txt

```diff
@@ -9031,16 +9031,21 @@ This command is used in all the item scripts for taming items. Running this
 command will make the pet catching cursor appear on the client connected to the
 invoking character, usable on the monsters with the specified pet ID number. It
 will still work outside an item script.
+If the <pet id> is PET_CATCH_UNIVERSAL it will return the monster target ID
+as long as it is in the pet_db.txt and does not have the MD_STATUS_IMMUNE monster mode.
+If the <pet id> is PET_CATCH_UNIVERSAL_ITEM it will return the monster target
+ID as long as it is in the pet_db.txt and the targeted monster requires the lure item used.
+See 'doc/mob_db_mode_list.txt' for more information about monster modes.
```

Comment:
@aleos89 
For both of these above:
"it will return" sounds odd to me, because this command does not return any value.
It would be better to refer to the catching process.
For example:
If the <pet id> is PET_CATCH_UNIVERSAL_ITEM the item will enable to catch the monster target as long as it is in the pet database and the targeted monster requires the lure item used.